test/mocha/20-tokens.js

```diff
@@ -1,18 +1,398 @@
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(result);
+      should.exist(err);
+      err.message.should.equal('"attributes" must be a Uint8Array.');
+    }
+  });
+  it('should throw error if attributes.length is greater than MAX_AAD_SIZE',
+    async function() {
+      const tokenCount = 5;
+      const internalId = 'aM6pup9s6XTaYTho';
+      const attributes = Uint8Array.from(new Set([1, 2, 3, 4, 5, 6, 7, 8, 9]));
+      let err;
+      let result;
+      try {
+        result = await tokens.create({internalId, attributes, tokenCount});
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(result);
+      should.exist(err);
+      err.message.should.equal('"attributes" maximum size is 8 bytes.');
+    });
+  it('should throw error if token does not exist in database',
+    async function() {
+      const tokenCount = 1;
+      const attributes = Uint8Array.from(new Set([1]));
+      const internalId = 'aM6pup9s6XTaYTho';
+      const requester = 'requester';
+      let err;
+      let result;
+      try {
+        const {tokens: tks} = await tokens.create(
+          {internalId, attributes, tokenCount});
+        const token = tks[0];
+        // intentionally clear tokenBatch database to remove token created
+        await cleanBatchDB();
+        result = await tokens.resolve({requester, token});
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(result);
+      should.exist(err);
+      err.name.should.equal('NotFoundError');
+      err.message.should.equal('Token not found.');
+    });
+  it('should resolve token to the party identified by "requester"',
+    async function() {
+      const tokenCount = 1;
+      const internalId = 'aM6pup9s6XTaYTho';
+      const attributes = Uint8Array.from(new Set([1]));
+      const requester = 'requester';
+      let err;
+      let result;
+      let tks;
+      try {
+        tks = await tokens.create(
+          {internalId, attributes, tokenCount});
+        const token = tks.tokens[0];
+        result = await tokens.resolve({requester, token});
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      areTokens(tks);
+      should.exist(result.pairwiseToken);
+    });
+  it('should resolve token when called twice with same "requester"',
+    async function() {
+      const tokenCount = 1;
+      const internalId = 'aM6pup9s6XTaYTho';
+      const attributes = Uint8Array.from(new Set([1]));
+      const requester = 'requester';
+      let err;
+      let result1;
+      let result2;
+      let tks;
+      try {
+        tks = await tokens.create(
+          {internalId, attributes, tokenCount});
+        const token = tks.tokens[0];
+        result1 = await tokens.resolve({requester, token});
+        // resolve token with same requester again
+        result2 = await tokens.resolve({requester, token});
+
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
```

Comment:
use `assertNoError(err)` and `assertNoError` should be added to the list of globals here: https://github.com/digitalbazaar/bedrock-tokenization/blob/initial/test/.eslintrc.js#L3